Grant AAD portal access by adding owners to a specific group

christeredvartsen opened the pull request:
Ref navikt/teams#72, we need to make sure group owners are in a specific group in AAD to gain access to the portal so that they can administer their groups.

Resolves #43

christeredvartsen committed:
Update dependencies and fix some issues raised by static analysis

christeredvartsen committed:
Use new version of actions/cache

christeredvartsen committed:
Bump PHP version in docker image

christeredvartsen committed:
Run static analysis on the entrypoint script

christeredvartsen committed:
Run statis analysis on the tests directory as well

Also fixed a bunch of issues as a result of running the analysis

christeredvartsen committed:
Also use psalm for static analysis

x10an14 reviewed (approved):
LGTM - not that I've pulled your code and tested myself, mind you!

christeredvartsen committed:
Dump all deps and fix all resulting incompatibilities

christeredvartsen committed:
Update deps once more

christeredvartsen committed:
Finally add the code that actually resolves #43

Also add some missing commas here and there, and a small refactor in the main Runner class.

christeredvartsen committed:
Bump deps

christeredvartsen committed:
Add slight improvement to validation of potentially missing/empty environment variables

christeredvartsen committed:
Add some missing commas
